### PR TITLE
Fix translations

### PR DESCRIPTION
**Pull request by ImUrX**

### CODE REVIEW

File: gui/public/i18n/vi/translation.ftl [hunk 1]
```diff
@@ -291,8 +291,8 @@ settings-general-gesture_control-description = Cho phép chạm vào tracker đ
 # This is a unit: 3 taps, 2 taps, 1 tap
 # $amount (Number) - Amount of taps (touches to the tracker's case)
 settings-general-gesture_control-taps = { $amount ->
-#   [one] 1 lần # Vietnamese doesn't have singular/plural difference but just leave it here to match up line count, easier for proofing with side by side English
-    *[other] { $amount } lần 
+    [one] 1 tap
```
Comment (Louka3000) [inline gui/public/i18n/vi/translation.ftl:294]:
```suggestion
    [one] 1 lần
```

File: gui/public/i18n/vi/translation.ftl [hunk 2]
```diff
@@ -523,7 +523,7 @@ onboarding-automatic_proportions-recording-steps-4 = Làm thêm các chuyển đ
 onboarding-automatic_proportions-recording-processing = Đang xử lí kết quả...
 # $time (Number) - Seconds left for the automatic calibration recording to finish (max 15)
 onboarding-automatic_proportions-recording-timer = { $time ->
-#   [one] còn lại 1 giây # Vietnamese doesn't have singular/plural difference but just leave it here to match up line count, easier for proofing with side by side English
+    [one] 1 second
```
Comment (Louka3000) [inline gui/public/i18n/vi/translation.ftl:526]:
```suggestion
    [one] còn lại 1 giây
```

**Review by Louka3000 — CHANGES_REQUESTED** [review]
Since there's no difference between singular and plural, you could just copy from the plural and replace the `{ $trackers/time }` by `1`

**Comment by ImUrX** [comment]
I kept the strings untranslated because they are not gonna get triggered

**Review by Louka3000 — APPROVED** [review]
Indeed :o

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]
Remove the German translations, we have  a PR already.

**Review by TheDevMinerTV — APPROVED** [review]

**Review by ButterscotchV — APPROVED** [review]

**Pull request merged**